Order_Car/styles/theme.css

```diff
@@ -0,0 +1,21 @@
+.yellow-theme {
+    --gray-hover-background: #7d827d;
+    --gray-background: #ffba08;
```

Comment:
please, don't put yellow color to a variable with name `--gray-background` ?
that was I explained 2 times on lectures